modules/tinymce/CHANGELOG.md

```diff
@@ -31,6 +31,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Notifications no longer require a timeout to disable the close button #TINY-6679
 - The editor theme is now fetched in parallel with the icons, language pack and plugins #TINY-8453
 - Calls to `editor.selection.setRng` now update the cursor position bookmark used when focus is returned to the editor #TINY-8450
+- Changed the default height of Editor from `200px` to `400px` #TINY-6860
```

Comment:
This should probably be in the `Improved` section, as it's being changed to make the editor more usable by default. Also `Editior` shouldn't be capitalised.

- [ ] Move this to Improved and reword to match

Reply:
It is already in Improved section

Reply:
Ahh sorry, it started with `Changed` and I couldn't see the top. In that case the line below should be in `Changed` instead and this needs rewording.